config/dspace_set_list.json

```diff
@@ -0,0 +1,1172 @@
+{
```

Comment:
I used the OAI-PMH ListSetSpecs verb to get the list of SetSpecs, but then I converted it into json so it would be easier to work with in Go. This code is all in a new branch of our OAI-PMH harvester repo (along with a couple other changes that make it easier to work with) that I intend to PR at some point. Which is all to say, this file should be easy to automatically update from the DSpace source.

Reply:
Added documentation for this.